auslib/web/common/history_all.py

```diff
@@ -47,55 +54,73 @@ def _get_histories(table, obj, process_revisions_callback=None):
                        ext={"exception": str(msg)})
 
 
-def get_rules_history():
+def rules_history():
     """GET /rules/history"""
     history_table = dbo.rules.history
-    return _get_histories(history_table, get_rules)
-
-
-def get_releases_history():
+    rules = _get_histories(history_table, get_rules)
+    history = {
+        'rules': rules,
+        'sc_rules': RuleScheduledChangeHistoryView().get_all()
```

Comment:
It looks like we're not apply any filters to this, which means we get unexpected results sometimes, eg: https://screenshots.firefox.com/zEoaUuLSSF7RyqXe/localhost

Let's fix this in a follow-up.